test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -1208,8 +1208,7 @@ public async Task GetTrustResultAsync_WithTimestampChainingToUntrustedRoot_NotAl
                 result.IsValid.Should().BeFalse();
                 resultsWithErrors.Count().Should().Be(1);
                 totalErrorIssues.Count().Should().Be(1);
-                totalErrorIssues.First().Code.Should().Be(NuGetLogCode.NU3028);
-                totalErrorIssues.First().Message.Should().Contain("A certificate chain processed, but terminated in a root certificate which is not trusted by the trust provider");
```

Comment:
Does this message changes between platforms?

Reply:
Thanks for reviewing! 
The X509ChainStatus consists of Status and StatusInformation. The StatusInformation varies on different platforms, but the Status keeps the same on different platform. 
We used to use StatusInformation for assertion, but now we would like to change to use Status. The PR making this change is https://github.com/NuGet/NuGet.Client/pull/3268